packages/wdio-logger/src/index.ts

```
@@ -215,6 +226,35 @@ getLogger.setLogLevelsConfig = (logLevels: Record<string, log.LogLevelDesc> = {}
         loggers[logName].setLevel(logLevel)
     })
 }
+getLogger.setMaskingPatterns = (maskingPatternsStringPerLoggerName: Record<string, string | undefined> = {}, wdioDefault:string|undefined = undefined) => {
```

Verify if, by default, we use `--key=[^ ]*` to mask the user's key from the command line. However, not sure when to apply it, always? Or only when Selenium, Sauce or BrowserStack are used?

---

Since I don't know how to check if a cloud provider is used, I suggest letting the user decide to configure their masking patterns. 
Instead, I provide helpful documentation on the `maskingPatterns` capabilities with an example on how to mask your cloud provider key